addons/pos_restaurant/static/src/app/floor_screen/floor_screen.xml

```diff
@@ -98,7 +98,7 @@
                         <div t-if="!activeTables?.length" class="empty-floor d-flex align-items-center justify-content-center h-100 fs-3 text-center text-muted" t-ref="map">
                             <span>Oops! No tables available.<br/>Add a new table to get started.</span>
                         </div>
-                        <div t-else="" t-ref="map" t-att-class="{'floor-kanban d-grid gap-3 p-3': pos.floorPlanStyle == 'kanban'}">
+                        <div t-else="" t-ref="map" class="h-100" t-att-class="{'floor-kanban d-grid gap-3 p-3': pos.floorPlanStyle == 'kanban'}">
```

Comment:
Thats wierd when we are in mobile mode 

![image](https://github.com/user-attachments/assets/6801ded7-f366-4f9f-971c-3995c7cb817f)


Can you maybe add the class conditionnaly ? Just check the floor plan mode

Reply:
oops. good catch!